docs/tutorials.rst

```diff
@@ -428,39 +441,41 @@ mildly following REST principles.
 .. code-block:: python
    :linenos:
 
-    import random
-    import string
+   import random
+   import string
+
+   import cherrypy
 
-    import cherrypy
 
-    @cherrypy.expose
-    class StringGeneratorWebService(object):
+   class StringGeneratorWebService(object):
+       exposed = True
```

Comment:
I missed this change in the previous review. It's ill-advised to make semantic changes simultaneous with whitespace edits. It's worse when they revert [intentional changes](/cherrypy/cherrypy/commit/5e1a48fdbfc5).

What's the motivation for this change?

To avoid another round on this, I'll restore the decorator after accepting this pull request, but please in the future try to make changes like this in separate commits if not separate pull requests.